main.cpp

```diff
@@ -35,319 +84,335 @@ send me a DM to check your pull request
  Wait for my code review.
  */
 
-/*
-1)knife
-5 properties:
-    1)pointiness
-    2)sharpness
-    3)thickness
-    4)length
-    5)price
-3 things it can do:
-    1)cut
-    2)stab
-    3)unscrew
- */
 
 struct Knife 
 {
-    // angle of knife point in degrees
+    
     float pointiness = 0.85f; 
-    // smallest thickness in mm
+ 
     float sharpness = 0.3f;
-    // biggest thickness
+
     float thickness = 1.5f;
-    //length in mm
+
     unsigned int length = 150;
-    //
+    
     float price = 75;
     
     struct Incision
     {
         float incisionDepth = 10;
         float incisionLength = 50;
```

Comment:
It is redundant to include the word 'incision' in the member variable names of a class named `Incision`
- [ ] revise